webscraping_scripts/UmutScrape/format.py

```diff
@@ -0,0 +1,124 @@
+""" 
+1.txt requirements:
+
+
+
+"""
+import re
+import string
+from spellchecker import SpellChecker
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize, sent_tokenize
+import nltk
+nltk.download('stopwords')
+# necessary installs:
+# pip install --user -U nltk
+# pip install pyspellchecker  
+# and the regex library
+
+
+def main():
+    f = open("1.txt", "r")
+    content = f.read()
+    # -------------- getting rid of the increasing numbers in the text ---------------------
```

Comment:
Let's think if there would be a reason to keep any numbers in the text. If not, it would be simpler to just remove all digits.

Reply:
Yes definitely, the algorithm I wrote for this is definitely unnecessary (and slow). I wanted to keep the numbers in the text just in case we need them.